niap-cc/Permissions/Companion/app/build.gradle

```diff
@@ -15,24 +15,45 @@
  */
 
 apply plugin: 'com.android.application'
-
+def applicationName = 'Companion'
 android {
-    compileSdkVersion 30
-
+    compileSdkVersion 33
+    signingConfigs {
+        companion {
+            storeFile file("../security/ec-p256.jks")
+            storePassword "android"
+            keyPassword "android"
+            keyAlias "platform"
```

Comment:
Nit: why is the alias for a non-platform signing key called platform? I thought the platform alias was reserved for the platform signing key of the device under test.

Reply:
okay. will change the alias name and update the corresponding documentation

Reply:
Update looks good to me; thanks!